sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@
 ### Features Added
 
 - Supported `userData` for `VirtualMachine` in create and update.
+- Fixed `actual and formal argument lists differ in length` exception in `VirtualMachineInner` class.
```

Comment:
remove this line, this does not relate to customer.

Reply:
Fixed in the new version.